mail/templates/product_order_receipt/body.html

```diff
@@ -30,7 +30,9 @@ <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px;
                 <strong style="font-weight: 700;">Quantity:</strong>  {{ line.quantity }}<br>
                 <strong style="font-weight: 700;">Unit Price:</strong> ${{ line.price }}<br>
                 <strong style="font-weight: 700;">Discount:</strong> ${{ line.discount }}<br>
-                <strong style="font-weight: 700;">Tax:</strong> ${{ line.tax_paid|floatformat:2 }}<br>
+                {% if line.tax_paid != "0.00" %}
```

Comment:
I think we have kind of made it hardcoded, We should use `>0` instead.

Reply:
line.tax_paid is a string. If we want an integer comparison, we would need to change the API. Should I?

Reply:
I think we should. I'm not expecting any other value in that string other then an int or float or decimal?